packages/table/src/css/index.ts

```diff
@@ -15,7 +16,9 @@ const cellPadding = `${layout.spacingXSmall} ${layout.spacingSmall}`
 const dark = {
   className: `.psds-theme--${themeNames.dark}`,
 
-  borderColor: colorsBorder.lowOnDark,
+  // TODO: should be using colorsBorder.lowOnDark but border collapse used with
```

Comment:
What's the TODO here?  Reads to me as more of a long-term note because it's something weird we apparently have to do.  When/how do you plan to come back?

Reply:
yes it should be a NOTE. i'll update